### PR TITLE
Update docker repo, add CI process

### PR DESCRIPTION
**Pull request by wilsonwu**

### CODE REVIEW

**Comment by wilsonwu** [comment]
make build problem need to be fixed:
```
#24 3.799 internal/route/routeRegister.go:6:2: ambiguous import: found package github.com/***/***/api/***/v1 in multiple modules:
#24 3.799 	github.com/***/*** (/workspace/api/***/v1)
#24 3.799 	github.com/***/***/api v0.0.0-00010101000000-000000000000 (/workspace/api/***/v1)
#24 3.799 internal/service/demo.go:9:2: ambiguous import: found package github.com/***/***/api/common/v1 in multiple modules:
#24 3.799 	github.com/***/*** (/workspace/api/common/v1)
#24 3.799 	github.com/***/***/api v0.0.0-00010101000000-000000000000 (/workspace/api/common/v1)
#24 3.799 internal/service/demo.go:11:2: ambiguous import: found package github.com/***/***/api/general/v1 in multiple modules:
#24 3.799 	github.com/***/*** (/workspace/api/general/v1)
#24 3.799 	github.com/***/***/api v0.0.0-00010101000000-000000000000 (/workspace/api/general/v1)
#24 3.803 make: *** [Makefile:3: build] Error 1
```

**Comment by wilsonwu** [comment]
@yangyy93 please help to review

**Comment by yangyy93** [comment]
close #4

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+      - name: Set up buf tool
+        uses: bufbuild/buf-setup-action@v1.40.0
+        with:
+          version: latest
+
+      - name: Generate API
+        run: make api_gen
+
+#      - name: Run golangci-lint
```
Comment (yangyy93) [inline .github/workflows/main.yml]: Can the comments here be released?

**Review by yangyy93 — APPROVED** [review]

**Pull request merged**